src/navinator.js

```diff
@@ -25,26 +26,35 @@
             var _ = this;
             _.defaults = {
                 buttonElement: '#navinator-button',
+                debug: false,
                 direction: 'left',
                 minWidth: 0,
-                navZ: 'bottom',
-                pageElement: '#page-container'
+                pageElement: '#page-container',
+                subNavElement: false
```

Comment:
Added a default state of `false` to test against whether the user wants to use sub navigation or not. If wanted, the user would add an optional string with the `class` to use as sub navigation elements.